app/visualisation_methods/interaction/plugin.rb

```diff
@@ -26,7 +26,7 @@ def handler
   end
 
 
-  def getInteraction( param1, param2, outputParam)
+  def getInteraction(param1, param2, outputParam)
```

Comment:
Jeśli już refaktorujemy - zmień nazwę na get_interaction. We wszystkich źródłach ruby'ego używamy snake case dla nazw metod.